local-cli/runIOS/runIOS.js

```diff
@@ -65,18 +87,46 @@ function runIOS(argv, config, args) {
   child_process.spawnSync('xcrun', ['simctl', 'launch', 'booted', bundleID], {stdio: 'inherit'});
 }
 
-function matchingSimulator(simulators, simulatorName) {
-  for (let i = simulators.length - 1; i >= 0; i--) {
-    if (simulators[i].name === simulatorName || formattedSimulatorName(simulators[i]) === simulatorName) {
-      return simulators[i];
+function runOnDevice(selectedDevice, scheme, xcodeProject){
+  buildProject(xcodeProject, selectedDevice.udid, scheme);
+  const iosDeployInstallArgs = [
+    '--bundle', 'build/Build/Products/Debug-iphoneos/' + scheme + '.app',
+    '--id' , selectedDevice.udid,
+    '--justlaunch'
+  ];
+  console.log('installing and launching app on connected device...');
+  child_process.spawnSync('ios-deploy', iosDeployInstallArgs, {stdio: 'inherit'});
```

Comment:
Good idea, i've added a message if ios-deploy is not installed globally:
![failed](https://cloud.githubusercontent.com/assets/9102810/18003116/19bbbb94-6b8c-11e6-9a59-f1904de9a6b1.png)
![succeeded](https://cloud.githubusercontent.com/assets/9102810/18003115/19ba14f6-6b8c-11e6-8992-0bfd78162c9d.png)